portality/scripts/remove_heloise_policy.py

```diff
@@ -0,0 +1,38 @@
+import csv
+
+from portality.models import Journal
+from portality.settings import BASE_FILE_PATH
+
+if __name__ == "__main__":
+
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("-o", "--out", help="output file path")
+    args = parser.parse_args()
+
+    if not args.out:
+        args.out = "out.csv"
+        # print("Please specify an output file path with the -o option")
+        # parser.print_help()
+        # exit()
+
+    with open(args.out, "w", encoding="utf-8") as f:
+        writer = csv.writer(f)
+
+        f = open(BASE_FILE_PATH + "/scripts/resources/journalID-for-heloise-removal.txt")
+        ids = f.readlines()
+
+        for i in ids:
+            j = Journal.pull(i.rstrip('\n'))
+            print(j.bibjson().deposit_policy)
+            if j:
+                bib = j.bibjson()
+                deposit_policy = j.bibjson().deposit_policy
+                if 'Héloïse' in deposit_policy:
+                    deposit_policy.remove('Héloïse')
+                if len(deposit_policy) == 0:
+                    bib.has_deposit_policy = False
+                    j.save(blocking=True)
+                    writer.writerow("Journal {} modified and saved.".format(j.id))
```

Comment:
Ah yeah, here's your problem with the output file format.  You need to put the row as a list:

```
writer.writerow(["Journal {} modified and saved.".format(j.id)])
```

Reply:
Yep, I realised right after your comment :)